scrapers/vandora

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+# $1 determines what to be scraped
+# if empty then it will scrape get all hospital data
+# o - O2 supply
+# i - ICU beds
+# m - mild symptomatic case beds
+
+case $1 in
+	'')	tid=1;;
```

Comment:
change tid to 13. The '3' matters because then it will show only relevant data i.e. the hospitals that have vacant beds. Using only 1 will pull all the data, including the ones that are full.

Reply:
If we get everything then we can query for the ones with beds > 0.
If someone wants to check the nearest hospital having beds then that might be useful.